### ISSUE TITLE
New requests for about this digital text box

### ISSUE THREAD

**Issue by davidamichelson** [srophe/syriac-corpus-app#129]
@jedwardwalters Please review and if you approve re-assign to W

For this section:
Source:
G. H. Gwilliam, J. Pinkerton and John Gwynn (eds.), The New Testament in Syriac [ܕܝܬܝܩܐ ܚܕܬܐ ܕܡܪܢ ܝܫܘܥ ܡܫܝܚܐ] (London: British and Foreign Bible Society, 1905).
->It would be good to have a link here to the Syriaca.org bibl module entry for this item. Perhaps by including the small book icon or the Syriaca.org icon like we do in the footnotes of the Syriaca modules?

Type of Text: Print Transcription
Status: Uncorrected Transcription

->both of these need [?] buttons that can link to documentation or else provide a definition of these terms?

**Comment by jedwardwalters** [comment]
Yes, I agree. I was actually wondering about the best way to provide some kind of link to documentation for those things. And I also agree that the small book icon would look nice.

Circling back to another issue on your source, though: do we need the parentheses around eds?

**Comment by davidamichelson** [comment]
I added the parens thing to the long how to cite issue :)

**Comment by jedwardwalters** [comment]
Okay, then I'll go ahead and assign these changes over to @wsalesky

**Comment by wsalesky** [comment]
Is there documentation to link to for: 
Type of Text: Print Transcription
Status: Uncorrected Transcription

When there is I will add the link. Working on the other part today.

**Comment by wsalesky** [comment]
@jedwardwalters @davidamichelson Can you highlight (screen shot) where the icon/link to Syriac.org  record should appear? with the "Source: " ?

**Comment by jedwardwalters** [comment]
@wsalesky I was about to do this, but it looks like the dev server is down.

**Comment by wsalesky** [comment]
It is just the corpus, unfortunately.
-Winona

**Comment by wsalesky** [comment]
@jedwardwalters Checking back in on this one.

**Comment by jedwardwalters** [comment]
@wsalesky I'll work on the documentation ASAP.

As for the location of the icon/link, it might work best to put it on the same line as the "Source:" heading, but justified to the right:
<img width="325" alt="screen shot 2018-05-07 at 9 41 04 am" src="https://user-images.githubusercontent.com/11543800/39704988-0eddd90a-51db-11e8-98e4-f73eb85adc83.png">

But maybe @davidamichelson has a different opinion?

**Comment by wsalesky** [comment]
@jedwardwalters Added. Let me know what you think.

**Comment by jedwardwalters** [comment]
I like it. We'll see if Dave or Dan have any thoughts at our meeting.

**Comment by wsalesky** [comment]
Moved icon.

**Comment by jedwardwalters** [comment]
Looks good to me.

**Comment by davidamichelson** [comment]
FYI, the period at the end of the source still needs adding. See http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/100

Source:
G. H. Gwilliam, J. Pinkerton and John Gwynn eds., The New Testament in Syriac [ܕܝܬܝܩܐ ܚܕܬܐ ܕܡܪܢ ܝܫܘܥ ܡܫܝܚܐ] (London: British and Foreign Bible Society, 1905) Link to Syriaca.org Bibliographic Record

**Comment by jedwardwalters** [comment]
@wsalesky It appears to be working on almost all the records where expected. For some reason, though, it's not showing up on texts 1-23.

http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/1.html

**Comment by wsalesky** [comment]
That has been fixed. There remains a space between the end of the citation and the period that I am having a very difficult time eliminating. Leaving this open.

**Comment by davidamichelson** [comment]
Hmm, there is still no period in the box for record 100 see
<img width="389" alt="screenshot 2018-05-10 09 41 25" src="https://user-images.githubusercontent.com/3083773/39875560-89b65f1e-5436-11e8-85ba-16e19a4475e9.png">

**Comment by jedwardwalters** [comment]
Interesting. It was showing up earlier when I checked that record, but not now.

**Comment by wsalesky** [comment]
Sorry all, I'm trying to get rid of that extra space, it is causing all sorts of issues.

**Comment by davidamichelson** [comment]
:)

**Comment by wsalesky** [comment]
@davidamichelson  @jedwardwalters  Okay, I think I have it working on dev. Please review, I used the following to review: (if you know there are citations with different structures check those as well)
http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/26 (manuscript)
http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/100 (monograph)
http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/1 (monograph in series)

**Comment by jedwardwalters** [comment]
Looks good in all the places I checked it.

**Comment by davidamichelson** [comment]
👍
